### PR TITLE
[Docs] PR for Incorrect information in the 'Configure the Yugabyte Platform' documentation section

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://github.com/yugabyte/yugabyte-db/issues/11967

Also, edits for clarity and formatting changes

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 1]
```diff
@@ -171,10 +140,10 @@ For each node you want to add, click **Add Instances** to add a YugabyteDB node.
...
 To provision your nodes manually, you have the following two options:
 
-1. If the SSH user you provided has sudo privileges but requires a password, you can [run the pre-provisioning script](#running-the-pre-provisioning-script).
+1. If the SSH user you provided has sudo privileges but requires a password, you can [run the preprovisioning script](#running-the-preprovisioning-script).
 2. If the SSH user does not have any sudo privileges, you need to [set up the database nodes manually](#setting-up-database-nodes-manually).
 
-#### Running the pre-provisioning script
+#### Running the preprovisioning script
 
 This step is only required if you set **Manually Provision Nodes** to true and the SSH user has sudo privileges which require a password; otherwise you skip this step.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:149]: another 'pre-provisioning' below

**Review by ddhodge — APPROVED** [review]

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 2]
```diff
@@ -73,81 +73,50 @@ Configuring the on-premises provided consists of a number of steps.
...
+- Enable the **Air Gap Install** field if you want the installation to run in an air-gapped mode without expecting any internet access.
 
-#### Node Exporter User
+- Optionally, you may enable **Advanced** and complete the following: 
 
-Override the default Prometheus user. This is useful when the user is pre-provisioned on nodes (in case user creation is disabled). If overridden, the installer checks whether or not the user exists and creates the user if it does not exist.
+  - Use the **Desired Home Directory** field to specify the home directory of the `yugabyte` user. The default value is `/home/yugabyte`.
+  - Use the **Node Exporter Port** field to specify the port number for the node exporter. The default value is 9300.
+  - Enabe the **Install Node Exporter** if you want the node exporter installed. You can skip this step if you have node exporter already installed on the nodes. Ensure you have provided the correct port number for skipping the installation.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:106]: Oh, I meant "Enable **Install Node Exporter** if you want the node exporter installed. "

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 3]
```diff
@@ -73,81 +73,50 @@ Configuring the on-premises provided consists of a number of steps.
...
-Define the disk volume of a node.
-
-#### Mount Paths
-
-Define a mount point with enough space to contain your node density. Use `/data`. If you have multiple drives, add these as a comma-separated list, such as, for example, `/mnt/d0,/mnt/d1`.
+- Use the **Machine Type** field to define a value to be used internally as an identifier in the **Instance Type** universe field.
+- Use the **Num Cores** field to define the number of cores to be assigned to a node.
+- Use the **Mem Size GB** field to define the memory allocation of a node.
+- Use the **Vol size GB** field to define the disk volume of a node.
+- Use the **Mount Paths** field to define a mount point with enough space to contain your node density. Use `/data`. If you have multiple drives, add these as a comma-separated list, such as, for example, `/mnt/d0,/mnt/d1`.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:119]: Good catch. It's an error on the screen shot. Should be /data

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 4]
```diff
@@ -73,81 +73,50 @@ Configuring the on-premises provided consists of a number of steps.
...
 
 ### Define regions and zones
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:122]: Fixed

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md (hunk 1 above)
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:149]: Fixed

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md [hunk 5]
```diff
@@ -73,81 +73,50 @@ Configuring the on-premises provided consists of a number of steps.
...
-
-Define the disk volume of a node.
-
-#### Mount Paths
-
-Define a mount point with enough space to contain your node density. Use `/data`. If you have multiple drives, add these as a comma-separated list, such as, for example, `/mnt/d0,/mnt/d1`.
+- Use the **Machine Type** field to define a value to be used internally as an identifier in the **Instance Type** universe field.
+- Use the **Num Cores** field to define the number of cores to be assigned to a node.
+- Use the **Mem Size GB** field to define the memory allocation of a node.
+- Use the **Vol size GB** field to define the disk volume of a node.
```
[the remark this answers is not in the document]
Comment (lizayugabyte) [reply docs/content/preview/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/on-premises.md:118]: Fixed

**Pull request merged**